baselines/dom.generated.d.ts

```diff
@@ -13668,6 +13668,15 @@ interface ClipboardEventInit extends EventInit {
 interface IDBArrayKey extends Array<IDBValidKey> {
 }
 
+interface HTMLFileInputElement extends HTMLInputElement {
+    files: FileList;
+}
+
+declare var HTMLFileInputElement: {
```

Comment:
this introduces a new value `HTMLFileInputElement` that does not exisit any where.